rero_ils/modules/stats/api.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+# Copyright (C) 2020 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Stats for pricing records."""
+import arrow
+from dateutil.relativedelta import relativedelta
+from invenio_search.api import RecordsSearch
+
+from ..acq_order_lines.api import AcqOrderLinesSearch
+from ..documents.api import DocumentsSearch
+from ..items.api import ItemsSearch
+from ..libraries.api import LibrariesSearch
+from ..loans.logs.api import LoanOperationLog
+from ..patrons.api import PatronsSearch
+
+
+class StatsForPricing:
+    """Statistics for pricing."""
```

Comment:
Sometimes `stats` and sometimes `statistics`. Maybe it could be the same everywhere. But it's not so much important.